.github/workflows/scripts/biome_disable_check.py

```
@@ -58,25 +56,25 @@
         return False
 
 
-def check_eslint(files_or_directories):
+def check_biome(files_or_directories):
```

Ruff (0.8.2)

`check_biome` is too complex (11 > 10)

(C901)